sdks/python/apache_beam/runners/interactive/extensions/interactive_beam_side_panel/package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "interactive-beam-side-panel",
+  "version": "0.1.0",
+  "description": "A side panel providing information of current interactive environment, data sets associated with PCollections, pipeline state, job status and source capture.",
+  "keywords": [
+    "jupyter",
+    "jupyterlab",
+    "jupyterlab-extension"
+  ],
+  "homepage": "https://github.com/apache/beam",
+  "bugs": {
+    "url": "https://github.com/apache/beam/issues"
+  },
+  "license": "BSD-3-Clause",
+  "author": "ningk",
+  "files": [
+    "lib/**/*.{d.ts,eot,gif,html,jpg,js,js.map,json,png,svg,woff2,ttf}",
+    "style/**/*.{css,eot,gif,html,jpg,json,png,svg,woff2,ttf}"
+  ],
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "style": "style/index.css",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/apache/beam.git"
+  },
+  "scripts": { 
+    "build": "jlpm run build:lib",
+    "build:labextension": "cd interactive_beam_side_panel && rimraf labextension && mkdirp labextension && cd labextension && npm pack ../..",
+    "build:lib": "tsc",
+    "build:all": "jlpm run build:labextension",
+    "clean": "jlpm run clean:lib",
+    "clean:lib": "rimraf lib tsconfig.tsbuildinfo",
+    "clean:labextension": "rimraf interactive_beam_side_panel/labextension",
+    "clean:all": "jlpm run clean:lib && jlpm run clean:labextension",
+
+    "eslint": "eslint . --ext .ts,.tsx --fix",
+    "eslint:check": "eslint . --ext .ts,.tsx",
+    "prepare": "jlpm run clean && jlpm run build",
+    "watch": "tsc -w"
+  },
+  "dependencies": {
```

Comment:
I do not know much about js dependencies. Is it the normal standard to use a lower bound version only for dependencies?

Reply:
npm uses [semantic versioning](https://docs.npmjs.com/about-semantic-versioning), 
```
npm uses the tilde (~) and caret (^) to designate which patch and minor versions to use respectively.

So if you see ~1.0.2 it means to install version 1.0.2 or the latest patch version such as 1.0.4. If you see ^1.0.2 it means to install version 1.0.2 or the latest minor or patch version such as 1.1.0.
```